clients/imodelhub/src/imodelhub/Events.ts

```diff
@@ -180,25 +181,25 @@ export class CodeEvent extends BriefcaseEvent {
 }
 
 /** Sent when all [Code]($common)s for a [[Briefcase]] are deleted. Can occur when calling [[CodeHandler.deleteAll]] or [[BriefcaseHandler.delete]].
- * @alpha Hide Code API while focused on readonly viewing scenarios
+ * @public

```

Comment:
internal for consistency with AllLocksDeleted.

Reply:
Updated both AllCodesDeletedEvent and BriefcaseDeletedEvent to be internal for now, until there's a use case for them.